MAINTENANCE.md

```diff
@@ -0,0 +1,80 @@
+# Roles, People, and Processes for Maintaining the Tagging Schema
+
+This document outlines how this project is maintained.
+
+## Roles & People
+
+### Maintainer Role
+
+[Martin](https://github.com/tyrasd) maintains this project as part of his work on the iD Editor project.
+
+The maintainer role includes:
+- Having the final say in decisions for the tagging schema.
+- Creating releases.
+- Updating dependencies.
+
+and all the following roles.
+
+### Contributor Role
+
+- [Kyle](https://github.com/k-yle) contributes to this project as a volunteer.
+- [Tobias](https://github.com/tordans) contributes to this project as a volunteer.
+
+The contributor role includes:
+- Reviewing PRs.
+- Merging "clear-cut" PRs.
+
+and all the following roles.
+
+### Triage Role
+
+(No one has the triage role at the moment. Please reach out if you want to help.)
+
+The triage role includes:
+- Helping to clarify issues and PRs.
+- Closing issues as duplicates or not planned.
+
+### Contributors
```

Comment:
Note confusion with "Contributor role"

Reply:
I've renamed the above role to "Co-Maintainers".